Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -3504,6 +3504,15 @@ def gen2021HiMix(fragment,howMuch):
                                       '--geometry' : geom
                                       }
 
+    upgradeStepDict['NanoDQM'][k] = {'-s':'NANO,DQM:@nanoAODDQM',
```

Comment:
what is the rational to define the NanoDQM here ?
I do not see the miniAODDQM

Reply:
Thanks! I have updated the PR accordingly.